deno.jsonc

```diff
@@ -7,7 +7,7 @@
     "semiColons": false
   },
   "tasks": {
-    "test": "deno test --allow-net --allow-read --coverage=coverage --parallel",
+    "test": "deno test --allow-net --allow-all --coverage=coverage --parallel",
```

Comment:
`--allow-all` is insecure to use. if you meant to add a permission add it via `--allow-{perm}`